CI flake in app-tests TestJournalistLayout

Seeing this occur repeatedly today; re-running does not appear to resolve consistently.

Example test failure: https://circleci.com/gh/freedomofpress/securedrop/40492#tests/containers/1

Example output:

<details>
<summary>Click to expand</summary>

```
=================================== FAILURES ===================================
__________ TestJournalistLayout.test_account_edit_hotp_secret[en_US] ___________

self = <tests.pageslayout.test_journalist.TestJournalistLayout object at 0x7fbd161422b0>

    def test_account_edit_hotp_secret(self):
        self._journalist_logs_in()
        self._visit_edit_account()
>       self._visit_edit_hotp_secret()

/home/circleci/project/securedrop/tests/pageslayout/test_journalist.py:36: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/home/circleci/project/securedrop/tests/functional/journalist_navigation_steps.py:744: in _visit_edit_hotp_secret
    "Reset two-factor authentication for security keys like Yubikey")
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <tests.pageslayout.test_journalist.TestJournalistLayout object at 0x7fbd161422b0>
otp_type = 'hotp'
tooltip_text = 'Reset two-factor authentication for security keys like Yubikey'

    def _visit_edit_secret(self, otp_type, tooltip_text=''):
        reset_form = self.wait_for(
            lambda: self.driver.find_element_by_id("reset-two-factor-" + otp_type)
        )
        assert "/account/reset-2fa-" + otp_type in reset_form.get_attribute("action")
        reset_button = self.driver.find_elements_by_css_selector(
            "#button-reset-two-factor-" + otp_type)[0]
    
        # 2FA reset buttons show a tooltip with explanatory text on hover.
        # Also, confirm the text on the tooltip is the correct one.
        reset_button.location_once_scrolled_into_view
        ActionChains(self.driver).move_to_element(reset_button).perform()
        time.sleep(1)
        explanatory_tooltip_opacity = self.driver.find_elements_by_css_selector(
            "#button-reset-two-factor-" + otp_type + " span")[0].value_of_css_property("opacity")
        explanatory_tooltip_content = self.driver.find_elements_by_css_selector(
            "#button-reset-two-factor-" + otp_type + " span")[0].text
    
>       assert explanatory_tooltip_opacity == "1"
E       AssertionError
```
</details>

--- a/securedrop/tests/functional/journalist_navigation_steps.py
+++ b/securedrop/tests/functional/journalist_navigation_steps.py
@@ -720,15 +720,20 @@ class JournalistNavigationStepsMixin:
         # Also, confirm the text on the tooltip is the correct one.
         reset_button.location_once_scrolled_into_view
         ActionChains(self.driver).move_to_element(reset_button).perform()
-        time.sleep(1)
-        explanatory_tooltip_opacity = self.driver.find_elements_by_css_selector(
-            "#button-reset-two-factor-" + otp_type + " span")[0].value_of_css_property("opacity")
-        explanatory_tooltip_content = self.driver.find_elements_by_css_selector(
-            "#button-reset-two-factor-" + otp_type + " span")[0].text
 
-        assert explanatory_tooltip_opacity == "1"
-        if not hasattr(self, "accept_languages"):
-            assert explanatory_tooltip_content == tooltip_text
+        def explanatory_tooltip_is_correct():
+            explanatory_tooltip = self.driver.find_element_by_css_selector(
+                "#button-reset-two-factor-" + otp_type + " span"
+            )
+
+            explanatory_tooltip_opacity = explanatory_tooltip.value_of_css_property("opacity")
+            assert explanatory_tooltip_opacity == "1"
+
+            if not hasattr(self, "accept_languages"):
+                assert explanatory_tooltip.text == tooltip_text
+
+        self.wait_for(explanatory_tooltip_is_correct)
+
         reset_form.submit()
 
         alert = self.driver.switch_to_alert()

